examples/example-breakglass-role.tf

```diff
@@ -0,0 +1,66 @@
+module "example-break-glass-PRD" {
+  source                   = "../github-workflow-roles"
+  github_organization_name = var.default_github_organization_name
+
+  # Providers block is used to indicate the accounts this pipeline IAM role should be created in
+  providers = {
+    aws = aws.Production
+  }
+
+  # Specify the least permissions required for break glass access
+  inline_policy = data.aws_iam_policy_document.example-pod-identity_Production_permissions.json
+
+
+  rvm_account_id = var.rvm_account_id
+
+
+  principal_type = "breakglass"
+
+  breakglass_user_alias = "johndoe"
+  breakglass_user_email = "jdoe@my-organization.com"
```

Comment:
I'm not a big fan of using actual ICANN top-level domains (eg. `.com`); my go-to is `test.lab` since `.lab` isn't allowable on the public internet. Maybe I'm being too picky.